README.rst

```diff
@@ -1,10 +1,23 @@
-# qp-knight-lab-processing
-
 |Build Status| |Coverage Status|
 
-General Knight Lab Admin Commands for sequence processing in Qiita.
+Knight Lab Admin Commands for Qiita. This is an Admin only plugin.
+
+This plugin provides the following commands:
+
+#. Sequence Processing Pipeline (SPP):
+
+This command converts the BCL to FASTQ files, does adaptor and host/human sequence filtering, summarizes results and uploads the resulting files to Qiita as a new preparation and artifact. This command is based on [mg-scripts](https://github.com/biocore/mg-scripts) and (metapool)[https://github.com/biocore/metagenomics_pooling_notebook/].
```

Comment:
maybe this should be changed to 'one or more new preparations and artifacts'? Since it can be more than one.